src/components/PlaylistRow.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+
+import PlaylistExporter from "./PlaylistExporter"
+
+class PlaylistRow extends React.Component {
+  exportPlaylist() {
+    PlaylistExporter.export(this.props.access_token, this.props.playlist);
+  }
+
+  renderTickCross(condition) {
+    if (condition) {
+      return <FontAwesomeIcon icon={['far', 'check-circle']} size="sm" />
+    } else {
+      return <FontAwesomeIcon icon={['far', 'times-circle']} size="sm" style={{ color: '#ECEBE8' }} />
+    }
```

Comment:
for simple if/else, the ternary condition is frequently used but it is a matter of taste

```ts
return condition ?
   <FontAwesomeIcon icon={['far', 'check-circle']} size="sm" /> :
   <FontAwesomeIcon icon={['far', 'times-circle']} size="sm" style={{ color: '#ECEBE8' }} />
```